packages/ppx_const/ppx_const.1.0/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Andi McClure <andi.m.mcclure@gmail.com>"
+authors: "Andi McClure <andi.m.mcclure@gmail.com>"
+homepage: "https://github.com/mcclure/ppx_const"
+bug-reports: "https://github.com/mcclure/ppx_const/issues"
+license: "Creative Commons Zero"
+tags: "syntax"
+dev-repo: "git://github.com/mcclure/ppx_const.git"
+substs: "pkg/META"
+build: ["ocaml" "pkg/build.ml" "native=%{ocaml-native}%" "native-dynlink=%{ocaml-native-dynlink}%"]
+build-test: ["ocamlbuild" "-classic-display" "-use-ocamlfind" "src_test/test_ppx_const.byte" "--"]
```

Comment:
It looks like you use `ocamlfind` here in the test, but it's not listed in the depends below.  You can either add just `ocamlfind`, or `ocamlfind {test}` if you only use it for tests and not for the main build.